src/ui/hooks/settings.ts

```diff
@@ -129,3 +136,27 @@ export function useDeleteUserApiKey() {
 
   return { deleteUserApiKey, loading, error };
 }
+
+export function useGetWorkspaceApiKeys(workspaceId: string) {
+  const { data, loading, error } = useQuery(GET_WORKSPACE_API_KEYS, {
+    variables: { workspaceId: Buffer.from(workspaceId, "base64").toString().substr(2) },
```

Comment:
These IDs are meant to be opaque, please don't try to parse them. If you leave this as-is, you can change the query to do
```
  query GetWorkspaceApiKeys($workspaceId: ID!) {
    node(id: $workspaceId) {
      ... on Workspace {
        apiKeys {
          id
          createdAt
          label
          scopes
        }
      }
    }
  }
```

Reply:
Thanks, @loganfsmyth! I had meant to note this in the PR since it seemed at best unconventional to have to decode the key.